### ISSUE TITLE
An error occurred while trying to update settings: Could not connect to database (?).

### ISSUE THREAD

**Issue by teck44** [opensupports/opensupports#293]
Hello,
I have the same issue described here https://github.com/opensupports/opensupports/issues/282 when I tried to install the software in the step 3.
An error occurred while trying to update settings: Could not connect to database (?).

I have used the latest version of the software.
Can any one tell me how I can fix this issue please?

**Comment by mredigonda** [comment]
I assume you already tried clearing the contents of `api/config.php` and trying to install it again.
If so, could you please provide some details like:
1. What version of PHP and MySQL are you using?
2. What is your input in the form (for example: MySQL server: localhost, MySQL server port: empty, etc.)
Thank you for reporting!

**Comment by teck44** [comment]
Yes i have already cleaned the content of api/config.php (the file is empty now).
The version of Mysql that I have installed is mysql Ver 14.14 Distrib 5.7.23 and PHP 5.6.37-1

For the input, I have tried all possible combinaisons ( MySQL server:localhist, MySQL server port:empty, MySQL server port:3306, etc)

Ps: the connection to MySQL via the command line works normally.
Thank you for the response

**Comment by mredigonda** [comment]
Try replacing localhost with 127.0.0.1.

**Comment by mredigonda** [comment]
You probably would like to replace "localhost" by the IP of your MySQL server.
